src/nyc_trees/apps/users/views/group.py

```diff
@@ -45,13 +51,42 @@ def group_detail(request):
     show_mapper_request = group.allows_individual_mappers and \
         not user_is_group_admin(request.user, group)
 
+    tree_count = get_group_tree_count(group)
+
+    group_blocks = Territory.objects.filter(group=group)
+    group_blocks_count = group_blocks.count()
+
+    if group_blocks_count > 0:
+        completed_blocks = Survey.objects \
+            .filter(blockface__in=group_blocks) \
+            .distinct('blockface')
+        block_percent = "{:.1%}".format(
+            float(completed_blocks.count()) / float(group_blocks.count()))
+    else:
+        block_percent = "0.0%"
+
+    events_held = Event.objects.filter(group=group, ends_at__lt=now())
+    num_events_held = events_held.count()
+
+    num_event_attendees = EventRegistration.objects \
+        .filter(event__in=events_held) \
+        .filter(did_attend=True) \
+        .distinct('user') \
```

Comment:
We should ask Parks to confirm this method of attendance counting (unless they already have). It may be desirable to count "2" if a person attends two, distinct events. If we don't yet have a ruling, it need not block the PR.